Core/Dashboards/Models/GetExerciseSolutionResponseDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Core.Dashboards.Models
+{
+    public record GetExerciseSolutionResponseDto(
+        string title,
+        string description,
```

Comment:
Observe how it operates with a request on no description - alternatively make nullable